zebra-instructions.md

```diff
@@ -96,72 +153,112 @@ Install dependencies:
     sudo apt-get install redis
     redis-server
     ```
+
 2. Install and activate a node version manager (e.g. [`nodenv`](https://github.com/nodenv/nodenv#installation) or [`nvm`](https://github.com/nvm-sh/nvm#installing-and-updating))
 3. Install `boost` and `libsodium` development libraries
-   On Ubuntu:
+
     ```sh
     sudo apt install libboost-all-dev
     sudo apt install libsodium-dev
     ```
 
-Install `s-nomp`:
+#### Install `s-nomp`:
+
 1. `git clone https://github.com/ZcashFoundation/s-nomp`
 2. `cd s-nomp`
 3. Use the Zebra fixes: `git checkout zebra-mining`
 4. Use node 8.17.0:
+
     ```sh
     nodenv install 8.17.0
     nodenv local 8.17.0
     ```
-    or 
+
+    or
+
     ```sh
     nvm install 8.17.0
     nvm use 8.17.0
     ```
 
 5. Update dependencies and install:
+
     ```sh
     export CXXFLAGS="-std=gnu++17"
     npm update
     npm install
     ```
 
-Run `s-nomp`:
+## Run `s-nomp`:
+
 1. Edit `pool_configs/zcash.json` so `daemons[0].port` is your Zebra port
 2. Run `s-nomp` using `npm start`
 
 Note: the website will log an RPC error even when it is disabled in the config. This seems like a `s-nomp` bug.
 
 ## Install a CPU or GPU miner
 
-Install dependencies:
+### Arch-specific instructions
+
+We're going to install `nheqminer`, which supports various CPU and GPU Equihash
+solvers. We're going to focus on a CPU solver called `tromp` because that one is
+the easiest to install.
```

Comment:
This is not an Arch-specific intro, so it seems like it belongs in the heading above?

```suggestion
We're going to install `nheqminer`, which supports various CPU and GPU Equihash
solvers. We're going to focus on a CPU solver called `tromp` because that one is
the easiest to install.

### Arch-specific instructions
```

Reply:
This is actually specific to Arch, because the other instructions go with the default CPU solver, which is xenoncat.

nheqminer comes with tromp, xenoncat, and djezo. I can't run djezo since it seems to be written only in CUDA. I wasn't able to compile nheqminer with xenoncat either, and I didn't investigate the issue too much because I tried tromp, and it worked fine. That's why I use the following options to compile nheqminer:
`-DUSE_CUDA_DJEZO=OFF -DUSE_CPU_XENONCAT=OFF -DUSE_CPU_TROMP=ON`.

Reply:
It seems like tromp is the easiest solver to use.

Can we put it as the first alternative in the instructions, and say that the other solvers require specific hardware or configurations?

Reply:
Done in [a173e97](https://github.com/ZcashFoundation/s-nomp/pull/9/commits/a173e976f252fa254b62908d54a36e16b5e09e30).

Reply:
I also removed CUDA from the instructions for two reasons:

1. To shorted the instructions.
2. To avoid a sudden burst of blocks in case someone runs the miner on a fast GPU.